tiapp.xml

```diff
@@ -12,99 +11,101 @@
   <fullscreen>false</fullscreen>
   <navbar-hidden>false</navbar-hidden>
   <analytics>true</analytics>
```

Comment:
Perhaps put the analytics tag to false now as this depends on the backend being there :)